### ISSUE TITLE
Staff Application by Neil09

### ISSUE THREAD

**Issue by Neil09to** [PinoyLifeCraft/Pinoy-LifeCraft-Forums#31]
Please Change this format for Application Click here for the format #24 
Warning No slots for Mod,Helper...

**Comment by Neil09to** [comment]
Staff Application by Neil09

**Comment by PinoyLifeCraftModerator** [comment]
@Neil09to pls ediy your applyication format, use this format from #24

**Comment by Kaoruuu** [comment]
@Neil09to , Copy the #24 format and edit this issue post.

**Comment by Neil09to** [comment]
pano

**Comment by Neil09to** [comment]
kaoru

**Comment by Neil09to** [comment]
Name:Neil09
Can I apply to be a good Builder?
i want to be a builder because i will so many help players
My Real name is Neilivan Garcia my age is 10 years old
i will banned the troublemakers 30 minutes
yes/no but i have close freinds he/she
im Single
i will stay online everyday
yes im freindly
i will build in creative is a beutiful city

**Comment by Kaoruuu** [comment]
Again, suggested by the format.
-What would be your action if there's a troublemaker? (Your answer is good here)

-If he/she is a really close friend? (Your answer here is not on point, it is asking what if the trouble maker was your close friend? What will you do?, not  "IF you have any close friends or who are your friends"

**Comment by Neil09to** [comment]
Name:Neil09
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is Neilivan Garcia my age is 10 years old
ill banned all the troublemaker in 2 hours
yes my freinds is many he/she
im staying online everyday
im freindly beacause im so many freind there are so freindly to me

**Comment by Neil09to** [comment]
Name:Neil09
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is Neilivan Garcia my age is 10 years old
ill banned all the troublemaker in 2 hours
yes my freinds is TAPPY.SKT_Zed.Akatsuki1025.Jayson1.SPONGE.Pota_pepe2 and Ainz
im staying online everyday
im freindly beacause im so many freind there are so freindly to me

**Comment by Kaoruuu** [comment]
Noooo,
It doesn't matter which friend or if you describe the specific friends you have.
It is asking what will you do to the trouble maker if they were your close friends.

**Comment by Neil09to** [comment]
Name:Neil09
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is Neilivan Garcia my age is 10 years old
ill banned all the troublemaker in 2 hours if my close freind i will mute the player in 10 minutes
yes my freinds is TAPPY.SKT_Zed.Akatsuki1025.Jayson1.SPONGE.Pota_pepe2 and Ainz
im staying online everyday
im freindly beacause im so many freind there are so freindly to me i will mute player

**Comment by Kaoruuu** [comment]
"yes my freinds is TAPPY.SKT_Zed.Akatsuki1025.Jayson1.SPONGE.Pota_pepe2 and Ainz"
Remove that part, it's redundant to the format.

**Comment by Neil09to** [comment]
Name:Neil09
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is Neilivan Garcia my age is 10 years old
ill banned all the troublemaker in 2 hours if my close freind i will mute the player in 10 minutes
yes my freinds is many
im staying online everyday
im freindly beacause im so many freind there are so freindly to me i will mute playe

**Comment by Kaoruuu** [comment]
It doesn't ask how many friends you have. Try again and follow the format.

**Comment by Neil09to** [comment]
Name:Neil09
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is Neilivan Garcia my age is 10 years old
ill banned all the troublemaker in 2 hours if my close freind i will mute the player in 10 minutes
yes my freinds is 6
im staying online everyday
im freindly beacause im so many freind there are so freindly to me i will mute player

**Comment by Neil09to** [comment]
Name:Neil09
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is Neilivan Garcia my age is 10 years old
ill banned all the troublemaker in 2 hours if my close freind i will mute the player in 10 minutes
yes my freinds is many
im staying online everyday
im freindly beacause im so many freind there are so freindly to me i will mute player

**Comment by Neil09to** [comment]
Name:Neil09
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is Neilivan Garcia my age is 10 years old
ill banned all the troublemaker in 2 hours if my close freind i will mute the player in 10 minutes
im staying online everyday
im freindly beacause im so many freind there are so freindly to me i will mute player

**Comment by PinoyLifeCraftModerator** [comment]
Truth to be told you cannot just ban a troublemaker and builders cant ban u have to show us ur builds first

**Comment by CornNickel** [comment]
-If he/she is a really close friend?  "Pano kung kaibigan mo yung hacker,Gumagawa ng Masama anong gagawin mo?" 

Sana ma-gets mo na :)

**Comment by Poopeater** [comment]
Name:Poopeater
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 10 years
My Real Name is john irvin romay my age is 16 years old
ill banned all the troublemaker in 3 hours  if my close freind i will mute the player in 10 minutes
yes my freinds is many
im staying online everyday
im freindly beacause im so many freind there are so freindly to me i will mute player

**Comment by Poopeater** [comment]
sabi daw d2 ako mag aaply sa moderator o builder?

**Comment by HARLYPash0313** [comment]
Name:harlypasiosane
First of all Good day!!
I want to apply as a builder..
Because I want to build and also I want to help you guys promote your server,
and for the trouble makers I will warn them first, and if they make the same mistake again or they make
too much abuse to the server,
I will get them banned for maybe 2 to 3 weeks,
maybe thats enough for them to learn there lesson.
My real name is Harly Dean A. Pasiosane and Im 15 years old
yes I've got a lot of friends
Im staying online for maybe 5 to 6 hours almost everyday, except sunday. 
I've got a lot of friends simply because Im just friendly.
I hope that you'll appreciate my application.
THANK YOU.

**Comment by PinoyLifeCraftAdmin** [comment]
@HARLYPash0313 Pls make a new issue containing about ur application or go to this link : https://plcstaff.typeform.com/to/kMH5qz

**Comment by Poopeater** [comment]
Name:Poopeater
Can i apply to be a good Builder?
I want to be a builder beacause ill dont try to be a staff i help another people in this server and ill banned
a abusing players and hackers in 3 to 5 months
My Real Name is john irvin romay my age is 16 years old
ill banned all the troublemaker in 3 hours if my close freind i will mute the player in 10 minutes
yes my friends is many because I am friendly
im staying online everyday
im friendly beacause im so many friend there are so friendly to me i will mute player

**Comment by Poopeater** [comment]
Name:Poopeater
Hi Every one!!
I want to apply as a Good builder..
Because I want to build and also I want to help you guys our  server improve
and for the trouble makers I will warn them first, and if they make the same mistake again or they make
too much abuse to the server,
I will get them banned for maybe 2 to 3 weeks,
maybe thats enough for them to learn there lesson.
My real name is john irvin romay and Im 16 years old
yes I've got a lot of friends
Im staying online for maybe 5 to 6 hours almost everyday, except sunday.
I've got a lot of friend's simply because Im just friendly.
I hope that you'll appreciate my application.
Thanks!
